scripts/dts/extract_dts_includes.py

```diff
@@ -744,9 +737,9 @@ def main():
     get_aliases(dts['/'])
     get_chosen(dts['/'])
 
-    yaml_list = load_yaml_descriptions(dts, args.yaml)
+    extract.globals.yaml_list = load_yaml_descriptions(dts, args.yaml)
```

Comment:
This makes yaml_list a global with a hardly understandable variable name. What about using `bindings` instead?

Reply:
Agreed.